File: php/Disk.php
<?php
   
    class Disk{
        //atribut
        private $type = "";
        private $capacity = 0;
        private $price = 0;

        function __construct($type = 0, $capacity = 0, $price = 0){
            $this->type = $type;
            $this->capacity = $capacity;
            $this->price = $price;
        }

        //setter & getter
        function settype($type){
            $this->type = $type;
        }
        function gettype(){
            return $this->type;
        }

        function setcapacity($capacity){
            $this->capacity = $capacity;
        }
        function getcapacity(){
            return $this->capacity;
        }

        function setprice($price){
            $this->price = $price;
        }
        function getprice(){
            return $this->price;
        }

        function printDisk(){
            echo "Disk        : ".$this->gettype(). " ".$this->getcapacity(). " GB"."<br/>";
        }
        //destruktor
        function __destruct(){  
        }
    } 
?>

File: c++/disk.cpp
class Disk{
    private:
        //atribut
        string type;
        int capacity, price;

    public:
        //konstruktor
        Disk(){}  
        Disk(string t, int c, int p){
            this->type = t;
            this->capacity = c;
            this->price = p;
        }

        //setter & getter
        void setType(string t){
            this->type = t;
        }
        string getType(){
            return this->type;
        }

        void setCapacity(int c){
            this->capacity = c;
        }
        int getCapacity(){
            return this->capacity;
        }

        void setPrice(int p){
            this->price = p;
        }
        int getPrice(){
            return this->price;
        }

        void printDisk(){
            cout << "Disk        : " << this->type << " " << this->capacity << " GB" << endl;
        }

        //destruktor
        ~Disk(){
        }
};
